stacks/SiriAPIStack.ts

```diff
@@ -38,7 +38,7 @@ export const SiriAPIStack = ({ stack }: StackContext) => {
         defaults: {
             function: {
                 timeout: 20,
-                runtime: "nodejs18.x",
```

Comment:
We'll also need to add the runtime to the `NextjsSite` construct in the site stack or it'll keep using node 18 which is still the default

Reply:
done